default.nix

```diff
@@ -604,9 +605,9 @@ let overrideCabal = pkg: f: if pkg == null then null else lib.overrideCabal pkg
 in let this = rec {
   overrideForGhcjs = haskellPackages: haskellPackages.override {
     overrides = self: super: {
-      mkDerivation = drv: super.mkDerivation.override {
-        hscolour = ghc.hscolour;
-      } (drv // {
+      hscolour = ghc.hscolour;
```

Comment:
Hmm, I'm not sure this is so great.  Don't we want ghcjs derivations to use the ghc-built hscolour when using it as a tool, and provide the ghcjs-built hscolour when supplying it as a library?